src/api/v1/router.ts

```diff
@@ -18,61 +18,124 @@ router.get('/version', (req: Request, res: Response) => {
 });
 
 // CREATE NEW WALLET AND ENCRYPT
-router.post("/wallet", async (req: Request, res: Response) => {
+router.post('/wallet', async (req: Request, res: Response) => {
   try {
-    const {email, phone } = req.body
+    const { email, phone } = req.body;
     const wallet: IDecryptedWallet = createWallet();
-    const ethereumAddress = wallet.ethereumAddress
+    const ethereumAddress = wallet.ethereumAddress;
 
     if (email) {
       const encryptedWallet: IEncryptedWallet = await encryptWalletWithEmail(
-          ethereumAddress,
-          wallet.mnemonic,
-          wallet.privateKey,
-          email
-      )
+        ethereumAddress,
+        wallet.mnemonic,
+        wallet.privateKey,
+        email
+      );
 
       const data: IResponseWalletCreateEmail = {
         ...encryptedWallet,
         ethereumAddress,
         email
-      }
+      };
 
       return res.send({ data });
     }
 
     if (phone) {
       const encryptedWallet: IEncryptedWallet = await encryptWalletWithPhone(
-          ethereumAddress,
-          wallet.mnemonic,
-          wallet.privateKey,
-          phone
-      )
+        ethereumAddress,
+        wallet.mnemonic,
+        wallet.privateKey,
+        phone
+      );
 
       const data: IResponseWalletCreatePhone = {
         ...encryptedWallet,
         ethereumAddress,
         phone
-      }
+      };
 
       return res.send({ data });
     }
 
     const error: IResponseError = {
       message: 'No valid identity',
       code: ERROR_CODES.BAD_REQUEST
-    }
+    };
 
     return res.send({ error });
   } catch (err: any) {
-    loggingService.error("Could not create wallet", err.message);
+    loggingService.error('Could not create wallet', err.message);
 
     const error: IResponseError = {
       message: 'Could not create wallet',
       code: ERROR_CODES.INTERNAL_ERROR
-    }
+    };
+
+    return res.status(500).send({ error });
+  }
+});
+
+// CREATE NEW WALLET AND ENCRYPT FOR SERVICE WALLET
+router.post('/service/wallet', async (req: Request, res: Response) => {
+  try {
+    const { email } = req.body;
+    const wallet: IDecryptedWallet = createWallet();
```

Comment:
Instead of using an email address for service account wallets, we should probably use the account ID of the Labs API account that the account is being created for, and we should use this ID in place of the email both _inside_ and outside of the encrypted JSON (like the email field)

Reply:
Should paths be renamed to `/account/wallet`?

Reply:
I think the path is fine :)